doc/articles/toc.yml

```diff
@@ -65,6 +65,8 @@
           href: guides/uno-material-walkthrough.md
         - name: How to integrate Uno.Cupertino
           href: guides/uno-cupertino-walkthrough.md
+          href: guides/profiling-applications.md
+        - name: How to profile applications
```

Comment:
Doesn't really belong in how-tos... I would put it in the 'Development' section under 'Common tasks'